charts/lotus-fullnode/templates/deployment-secrets.yaml

```diff
@@ -22,8 +22,9 @@ subjects:
   name: {{ .Release.Name }}-secrets-writer
   namespace: {{ .Release.Namespace }}
 {{- end }}
-{{- if and .Values.secrets.libp2p.enabled (not .Values.secrets.libp2p.secretName) (not (lookup "batch/v1" "Job" .Release.Namespace "{{ .Release.Name }}-libp2p-secrets-creator")) }}
-
+{{- if and .Values.secrets.libp2p.enabled (not .Values.secrets.libp2p.secretName) }}
+{{- $secret_libp2p := lookup "v1" "Secret" .Release.Namespace "{{ .Release.Name }}-libp2p-secrets" -}}
```

Comment:
was assigning the lookup to a variable how to make it pass a truthiness check?
could you check for the job existence this way?

Reply:
I'll give it a try